frontends/web/src/routes/buy/info.tsx

```diff
@@ -76,7 +80,6 @@ export const BuyInfo = ({ code, accounts }: TProps) => {
     maybeProceed();
   }, [maybeProceed, options]);
```

Comment:
I find this `maybeProceed` a bit confusing, in the other places I assume we don't need a redirectAttempted.current ref because it's only called once in onSuccess?

Reply:
I'm not certain if it helps, but have you tried wrapping `requestKeystore` to be inside useCallback?